env/shot_step.yml

```diff
@@ -174,31 +184,40 @@ engines:
         location: {name: tk-multi-snapshot, type: app_store, version: v0.4.1}
         template_snapshot: max_shot_snapshot
         template_work: max_shot_work
-      tk-multi-workfiles:
+      tk-multi-workfiles2:
         allow_task_creation: true
+        custom_actions_hook: '{self}/custom_actions.py'
+        entities:
+        - caption: Assets
+          entity_type: Task
+          filters:
+          - [entity, type_is, Asset]
+          hierarchy: [entity.Asset.sg_asset_type, entity, step, content]
+        - caption: Shots
+          entity_type: Task
+          filters:
+          - [entity, type_is, Shot]
+          hierarchy: [entity.Shot.sg_sequence, entity, step, content]
```

Comment:
This chunk is seems to be repeated for all the app configurations. I'm guessing that's just the way the configs are organized, but thought I'd mention it just in case there's a way to define the app config once and then do some type of override per engine.